layers/poi/layer.sql

```diff
@@ -109,9 +102,29 @@ RETURNS TABLE(osm_id bigint, global_id text, geometry geometry, name text, name_
             osm_hash_from_imposm(osm_id) AS osm_id_hash,
             global_id_from_imposm(osm_id) as global_id
         FROM osm_poi_polygon
-            WHERE CASE WHEN bbox IS NULL THEN TRUE ELSE geometry && bbox END
-                AND zoom_level >= 14
+            WHERE zoom_level >= 14
                 AND (name <> '' OR (subclass <> 'garden' AND subclass <> 'park'))
         ) as poi_union
     ;
-$$ LANGUAGE SQL IMMUTABLE;
+$$ LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
+
+-- etldoc: layer_poi[shape=record fillcolor=lightpink, style="rounded,filled",
```

Comment:
Any reason to move this function at the bottom?

Reply:
`layer_poi` depends on the new `all_pois`, so they need to be defined in this order

Reply:
Oh I see!